### DISCUSSION TITLE
Question about currency calculator3

### DISCUSSION THREAD

**Discussion by starguo1998** [D3nan/Tafe24S1T102_Ext#5] [Q&A] [answered]
How can we integrate navigation logic to transition between the main menu page and the individual calculator pages while maintaining a consistent user experience?

**Comment by D3nan** [comment, marked as answer]
There are a few options. Let me know which ones we should include:
1. Include a "Return to Main Menu" button. 
2. We can define a fixed height and width for our calculator window. 
3. Agree on a specific font and font size

**Reply by starguo1998** [reply]
Hi Denan

Thank you for considering different options for integrating navigation logic while maintaining a consistent user experience. I think we can include all the options in our project to ensure a smooth transition between the main menu page and the individual calculator pages while maintaining consistency in user experience.

Xiaorui
